app/models/user.rb

```diff
@@ -25,7 +25,7 @@ class User < ActiveRecord::Base
   has_many :collection_roles, through: :identity_group
   has_many :user_seen_subjects
 
-  has_many :subject_queues, dependent: :destroy
```

Comment:
Yes its up to you. If we do want to set dependent relationships on the user model we should probably add Preferences and Memberships as well.